docs/iii.4.10-ldfld.md

```diff
@@ -10,7 +10,7 @@
 
 #### Description:
 
-The `ldfld` instruction pushes onto the stack the value of a field of _obj_. _obj_ shall be an object (type `O`), a managed pointer (type `&`), an unmanaged pointer (type `native int`), or an instance of a value type. The use of an unmanaged pointer is not permitted in verifiable code. _field_ is a metadata token (a `fieldref` or `fielddef` see [Partition II]#todo-missing-hyperlink)) that shall refer to a field member. The return type is that associated with _field_. `ldfld` pops the object reference off the stack and pushes the value for the field in its place. The field can be either an instance field (in which case _obj_ shall not be null) or a static field.
+The `ldfld` instruction pushes onto the stack the value of a field of _obj_. _obj_ shall be an object (type `O`), a managed pointer (type `&`), an unmanaged pointer (type `native int`), or an instance of a value type. The use of an unmanaged pointer is not permitted in verifiable code. _field_ is a metadata token (a `fieldref` or `fielddef` see [Partition II](ii.22.15-field-0x04.md)) that shall refer to a field member. The return type is that associated with _field_. `ldfld` pops the object reference off the stack and pushes the value for the field in its place. The field can be either an instance field (in which case _obj_ shall not be null) or a static field.
```

Comment:
> PDF links to II.22, but link a little deeper to the field table.

I appreciate that you took the time to investigate possible link targets. However, I'd like to request that you change this to a hyperlink to II.22 (`ii.22-metadata-logical-format-tables.md`), for two reasons:

1. We'd stay closer to the original material (like you mentioned, the PDF links to II.22). This is important because this project's primary goal is to translate the original document to the Markdown format; this project shouldn't try to "fix" the standard, even where it's inaccurate.

2. While II.22 doesn't explicitly define `fielddef` and `fieldref` (as mentioned e.g. here), it does define _FieldRef_ (as a token into the _MemberRef_ metadata table) as well as introduce the metadata tables (_Field_ among them), so in a way it covers both of those field references. Linking to II.22.15 OTOH would only cover _FieldDef_s but not _FieldRef_.

Reply:
Looks like you dug deeper ;) I'll update these links.

> this project's primary goal is to translate the original document to the Markdown format

Thanks for the clarity, this makes total sense! 

On a related tangent, have you seen this page: https://github.com/dotnet/runtime/blob/main/docs/design/specs/Ecma-335-Augments.md

Microsoft seems to have [moved away from updating specs](https://github.com/microsoft/TypeScript/issues/15711#issuecomment-661290678) so it might be interesting to incorporate the diffs that the dotnet team has decided on (maybe as a branch rebased ahead of `master`) 

Anyways, that's enough spitballing from me for now.

Reply:
I appreciate your input!

> On a related tangent, have you seen this page: https://github.com/dotnet/runtime/blob/main/docs/design/specs/Ecma-335-Augments.md

Yes, I've seen those kinds of "augment" documents before. I should add this one to the list (see the open issues at this repository). Thanks for mentioning it.

> Microsoft seems to have [moved away from updating specs](https://github.com/microsoft/TypeScript/issues/15711#issuecomment-661290678) so it might be interesting to incorporate the diffs that the dotnet team has decided on (maybe as a branch rebased ahead of master)

That's a fine idea. Adding their content to this project in dedicated branches would allow us to switch to a different branch and see the augmented/changed information, possibly marked up using a ` ```diff -... +... ``` ` so the changes are easier to spot.